libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/EvidenceRequest.java

```diff
@@ -10,19 +10,41 @@
 import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
 
 import java.util.Base64;
+import java.util.HashMap;
+import java.util.Map;
 
 @Getter
 @Builder
 @Jacksonized
 @AllArgsConstructor
 @ExcludeFromGeneratedCoverageReport
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class EvidenceRequest {
+    public static final String SCORING_POLICY_GPG45 = "gpg45";
+
     private static final ObjectMapper objectMapper = new ObjectMapper();
 
-    @JsonInclude(JsonInclude.Include.NON_NULL)
     private final String scoringPolicy;
+    private final Integer strengthScore;
+    private final Integer verificationScore;
+
+    // The JSON serialiser used by the Nimbus JWT library includes null values within claims so we
+    // need to have a way of giving it the values without including nulls. This is far from ideal.
```

Comment:
I think the Jackson lib should allow you to do this, which you have - have you tested if it needs this method with the `@JsonInclude(JsonInclude.Include.NON_NULL)` at the top of the class?
https://www.baeldung.com/jackson-ignore-null-fields#on-class